### PR TITLE
LLT-5140: Replace duration with u64 in udl

### PR DESCRIPTION
**Pull request by mathiaspeters**
### Problem
One of the NordVPN apps cannot currently use the duration type

### Solution
For now, we will use u64 in the interface, and convert it in libtelio

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by packgron — APPROVED** [review]
+2.0

File: crates/telio-model/src/features.rs [hunk 1]
```diff
@@ -1,11 +1,10 @@
 //! Object descriptions of various
 //! telio configurable features via API
 
-use std::{collections::HashSet, fmt, time::Duration};
+use std::{collections::HashSet, fmt};
 
 use num_enum::{IntoPrimitive, TryFromPrimitive};
 use serde::{de::IntoDeserializer, Deserialize, Deserializer, Serialize};
-use serde_with::{serde_as, DurationSeconds};
```
Comment (jjanowsk) [inline crates/telio-model/src/features.rs:8]: Please check if serde_with can be removed from the dependencies

**Review by jjanowsk — APPROVED** [review]
+1

File: crates/telio-model/src/features.rs (hunk 1 above)
Comment (mathiaspeters) [reply crates/telio-model/src/features.rs:8]: It still used in `telio-crypto`, but I removed it everywhere else

**Review by packgron — APPROVED** [review]

**Pull request merged**